### DISCUSSION TITLE
Practica 1 Semana 1 Modelamiento de Negocio

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#82] [Q&A]
Compañeros me podrían explicar esta parte de aquí, por lo que entiendo es poner un flujo de como funcionaria la pagina web o es sobre otra cosa?
![image](https://user-images.githubusercontent.com/88987587/155258794-7ce18364-bca7-44f7-98dc-564802fe1fe1.png)

**Comment by FabioMurillo16** [comment]
Hola Andrés, estas en lo correcto, el diagrama de flujo es en relación a la aplicación web, te adjunto mi diagrama para que te des una idea de como lo puedes hacer, si encuentras algunos fallos házmelo saber, saludos.
![Flowchart](https://user-images.githubusercontent.com/99093711/155261501-a40cd811-7576-4f64-8fe0-a2dc76754a4b.jpg)

**Reply by GloriaCoralCerecedo** [reply]
Hola, yo lo hice conforme al abogado osea como usará el sistema el abogado ya que dice que modelo de negocios

**Reply by AndresAVargas** [reply]
Muchas gracias compañero, disculpa llenaste las demás casillas después de eso, es que estoy un poco confundido en los requerimientos

**Reply by FabioMurillo16** [reply]
Si, mira aquí te dejo unos ejemplos de requerimientos de software de los cuales me apoye para resolverlo.
https://1library.co/document/y8x2r3wq-ejemplo-ieee-830.html
https://topicosbasededatos997954789.files.wordpress.com/2019/06/copia-de-ejemplo-formato-ieee-830.pdf

**Comment by tonydelfin** [comment]
![image](https://user-images.githubusercontent.com/98440007/155458164-6bc98602-4e94-4a20-b6ab-8802404db6e1.png)

¿Haremos todo el documento o solo hasta algún punto?

**Reply by GloriaCoralCerecedo** [reply]
Puedes quitar, poner lo que gustes. Igual utilizar otra plantilla
